STRUCTURE.md

```diff
@@ -67,14 +74,14 @@ address various expectations that Haskellers have, regarding things such as:
 
 Specifically, the following principles generally apply:
 
-* These libraries cannot depend on non-boot packages;
-* Abstraction should be kept minimal. `newtype`-wrapping a `ForeignPtr` is fine,
+* These libraries SHOULD NOT depend on non-boot packages;
+* Abstraction SHOULD be kept minimal. `newtype`-wrapping a `ForeignPtr` is fine,
   type class hierarchies without laws are not.
 * Whenever possible, if there are multiple secure ways to do something, without
   a definite 'this one is always better' option, users of these libraries should
   be given _all_ options, as well as a detailed explanation of why they should
   prefer any given one.
-* Partiality must not exist unless specifically marked as unsafe, through a
+* Partiality SHOULD NOT exist unless specifically marked as unsafe, through a
```

Comment:
I also feel this should be stronger. Partiality is unacceptable in 2022, and we already give a 'way out' by either naming or siloing.

Reply:
Yup', that's what the `SHOULD NOT` gives here :) I'm capitalising these terms in accordance with RFC2119, which provides the framework for  the exceptions that you've given already.